dashboard/app/controllers/api_controller.rb

```diff
@@ -11,6 +11,87 @@ def user_hero
     head :not_found unless current_user
   end
 
+  def update_lockable_state
+    updates = params.require(:updates)
+    updates.to_a.each do |item|
+      user_level_data = item[:user_level_data]
+
+      if user_level_data[:user_id].nil? || user_level_data[:level_id].nil? || user_level_data[:script_id].nil?
+        raise 'Must provide user, level, and script ids'
```

Comment:
I had assumed it would, but testing now it looks like we get back a 404, and the responseText contains the call stack (though I suspect that's only because this is a dev environment). Is there a better way of doing this such that we get back 500s?